src/olympia/reviewers/tests/test_utils.py

```diff
@@ -450,6 +450,27 @@ def test_actions_recommended_content_review(self):
             file_status=amo.STATUS_APPROVED,
             content_review=True).keys()) == expected
 
+    def test_actions_unlisted(self):
+        # Just regular review permissions don't let you do much on an unlisted
+        # review page.
+        self.version.update(channel=amo.RELEASE_CHANNEL_UNLISTED)
+        self.grant_permission(self.request.user, 'Addons:Review')
+        self.grant_permission(self.request.user, 'Addons:PostReview')
+        expected = ['reply', 'super', 'comment']
```

Comment:
it's expected that non `Addons:ReviewUnlisted` users can send a message regarding an unlisted version?

Reply:
Technically they can, but in reality they won't be able to access the review page.